STM32Cube/Core/Src/main.c

```diff
@@ -813,25 +825,16 @@ void can_handle_rx(const can_msg_t *message){
 void StartDefaultTask(void *argument)
 {
   /* USER CODE BEGIN 5 */
-	can_init_stm(&hfdcan1, can_handle_rx);
-	uint32_t LED_state = 0;
 	idx = 0;
 	/* Infinite loop */
 	for(;;)
 	{
-		/*sprintf ((char *)TxData, "CANTX%d", indx++);
-
-		if (HAL_FDCAN_AddMessageToTxFifoQ(&hfdcan1, &TxHeader, TxData)!= HAL_OK)
-		{
-		Error_Handler();
-		}*/
-		HAL_GPIO_WritePin(GPIOB, GPIO_PIN_5, LED_state);
-		LED_state = !LED_state;
-
 		can_msg_t message;
 		build_board_stat_msg(idx, E_NOMINAL, NULL, 0, &message);
-		can_send(&message);
-
+		if(xQueueSend(busQueue, &message, 1) != pdTRUE)
+		{
+			//Push a bus full error to the log queue
+		}
```

Comment:
missing line of code
`can_send(&message);`

Reply:
Fuck reviewable. I hate it so much.